.github/workflows/test-changed-firestore.yml

```diff
@@ -138,7 +140,7 @@ jobs:
         test-name: ["test:browser:nightly"]
     runs-on: ubuntu-latest
     needs: build
-    if: ${{ needs.build.outputs.changed == 'true'}}
+    if: ${{ github.event.workflow_run.trigger_source == 'workflow_dispatch' }}
```

Comment:
Would it be better to check if it's not `pull_request' in case we add new elements to `on:` above?

Reply:
Done.